### PR TITLE
Adding projectiles being fired and the character able to pick up pickups

### PR DESCRIPTION
**Pull request by TroyDexter**

### CODE REVIEW

**Review by Seanlegroulx — APPROVED** [review]
Approved!

**Review by markfilliph — APPROVED** [review]
Looks good. Approved.

**Review by ValeriaRincon — APPROVED** [review]
Looks good. Approved

File: Assets/Scripts/FinalBoss/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+public class Constants
+{
+    //Power up types
+    public const string Water = "Water";
+    public const string Fire = "Fire";
+    public const string Electricity = "Electricity";
+
+    //Pickup Types
+    public const int PickUpWater = 1;
```
Comment (DoopweeSLC) [inline Assets/Scripts/FinalBoss/Constants.cs:9]: may have been a good use of an enum here instead?

File: Assets/Scripts/FinalBoss/PlayerAim.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    void HandleMouseAim()
+    {
+        Vector3 mousePosition = GetMouseWorldPosition();
+
+        Vector3 aimDirection = (mousePosition - transform.position).normalized;
+        shootPosition = mousePosition;
+        float angle = Mathf.Atan2(aimDirection.y, aimDirection.x) * Mathf.Rad2Deg;
+        aimTransform.eulerAngles = new Vector3(0, 0, angle);
+
+        Debug.Log("Mouse Position: " + angle);
```
Comment (DoopweeSLC) [inline Assets/Scripts/FinalBoss/PlayerAim.cs:26]: remove debug logs before submitting to the main branch in the future

**Pull request merged**